lib/components/AlbumScreen/album_screen_content.dart

```diff
@@ -155,8 +157,10 @@ class _SongsSliverListState extends State<SongsSliverList> {
           // When user selects song from disc other than first, index number is
           // incorrect and song with the same index on first disc is played instead.
           // Adding this offset ensures playback starts for nth song on correct disc.
-          final indexOffset =
-              widget.childrenForQueue.indexOf(widget.childrenForList[0]);
+          final indexOffset = widget.parent.type == "MusicAlbum"
+              ? widget.childrenForQueue
+                  .then((_) => _?.indexOf(widget.childrenForList[0]) ?? 0)
```

Comment:
You should only use the underscore if you don't care about the actual value. Use something more descriptive otherwise :)
```suggestion
                  .then((childrenForQueue) => childrenForQueue?.indexOf(widget.childrenForList[0]) ?? 0)
```

Reply:
Oops bad habit